packages/blade/src/storybook-recipes/SimpleDashboard.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import type { Meta } from '@storybook/react';
+import { RecipeSandbox } from '~src/_helpers/storybook/Sandbox';
+import parameters from '~src/_helpers/storybook/recipeParameters';
+
+export function SimpleDashboard(): JSX.Element {
+  return (
+    <RecipeSandbox
+      title="Blade Dashboard"
+      codesandboxId="blade-dashboard-uyijy3"
+      activeFile="/src/Dashboard.tsx"
+      view="preview"
```

Comment:
@kamleshchandnani I added view="preview" here because the dashboard is looking better in full screen mode. You can drag and bring code or also open it in codesandbox with "open in CodeSandbox" link there. 🤔